metadata-ingestion/src/datahub/ingestion/source/sql/trino.py

```diff
@@ -248,6 +248,5 @@ def _parse_basic_datatype(s):
 
         return {"type": "null", "native_data_type": repr(s)}
 
-
```

Comment:
This seems to be the result of a newer black version. The build errors out otherwise